src/playback/qml/MuseScore/Playback/MixerPanel.qml

```diff
@@ -92,6 +103,8 @@ ColumnLayout {
                         }
 
                         prv.isPanelActivated = true
+                        let focusedIndex = item.channelItem.panel.order - 1000
```

Comment:
Sorry just one last thing - what does `1000` represent here? Is there another way to get this value?

Reply:
@mathesoncalum No worries,

Since the index doesn't change when navigating with the Tab key, the change is only in the order, which starts from 1000. You can verify this.

Reply:
Thanks - I was mainly wondering if there was an alternative way of pulling this value without hard-coding it. I think there is, but it would entail some overlap with #22050 (see the `navigationOrderStart` business in there).

@cbjeukendrup any thoughts on this? Do you reckon it's OK to leave this hard-coded for now and perhaps add a reminder to update it alongside #22050?

Reply:
How about simply `i`? Or am I saying something really stupid now?

Reply:
@cbjeukendrup Yeah I thought about this as well but sadly `i` doesn't correlate to the visual order of the channels (e.g. `i = 0` is the master channel, but the master channel isn't the first item in the visual order).

Reply:
That's weird... because `mixerPanelModel` is also used as the `model` for the `Mixer…Section`s below, and those don't seem to contain any reordering logic. So how does that work then?

Reply:
Ahh - I think we might have uncovered a bug in master here...

If the mixer is already open when the score is loaded, `setupConnections` gets a call before any instrument channel items have loaded (so the master channel is the only available item at `i = 0`). This method doesn't get another call unless you close/reopen the panel.

Reply:
So yeah, we should be able to use `i` but we'll first need to make sure `setupConnections` gets a call after all our instruments are loaded.